app/src/main/java/com/davidtomas/taskyapp/features/agenda/presentation/_common/components/Photos.kt

```diff
@@ -0,0 +1,238 @@
+        verticalArrangement = Arrangement.spacedBy(20.dp)
+    ) {
+        Spacer(modifier = Modifier.height(17.dp))
+        Text(
+            text = "Photos",
+            style = MaterialTheme.typography.titleMedium.copy(fontWeight = FontWeight.SemiBold)
+        )
+        if (photos.isEmpty()) {
+            Box(
+                modifier = Modifier.fillMaxWidth(),
+                contentAlignment = Alignment.Center
+            ) {
+                Row(
+                    modifier = Modifier
+                        .then(
+                            if (isEditable) Modifier
+                                .clickable {
+                                    singlePhotoPickerLauncher.launch(
+                                        PickVisualMediaRequest(ActivityResultContracts.PickVisualMedia.ImageOnly)
+                                    )
+                                } else Modifier,
+                        ),
+                    horizontalArrangement = Arrangement.Center,
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_add_icon),
+                        contentDescription = ""
+                    )
+                    Text(
+                        text = "Add photos"
+                    )
+                }
+            }
+        } else {
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth(),
+            ) {
+
+                val chunkedPhotosList = photos.chunked(5)
+
+                Column(verticalArrangement = Arrangement.spacedBy(8.dp)) {
+                    for (listOf5photos in chunkedPhotosList) {
+                        Row(
+                            horizontalArrangement = Arrangement.SpaceAround,
+                            modifier = Modifier.fillMaxWidth()
+                        ) {
+                            for (photo in listOf5photos) {
+                                AsyncImage(
+                                    model = photo,
+                                    contentDescription = "image description",
+                                    modifier = Modifier
+                                        .alpha(0.8f)
+                                        .size(60.dp)
+                                        .clip(RoundedCornerShape(5.dp))
+                                        .border(
+                                            border = BorderStroke(2.dp, Color.Gray)
+                                        )
+                                        .clickable {
+                                            onPhotoClicked(photo)
+                                        },
+                                    contentScale = ContentScale.FillBounds
+                                )
+                            }
+                            if (listOf5photos.size < 5 && isEditable) {
```

Comment:
This doesn't really make sense :D If your list definitely has 5 items as per the name, how can it have less than 5 items? I smell a bad naming 😄